### PR TITLE
Finished Google Homepage 

### PR DESCRIPTION
**Pull request by smeetthakkar**
This is the final version of the google homepage. Let me know if this is good to go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<p>The assigment was to recreate the Google Homepage. (the simple one with just a search box).</p>
+
+<p>Points to remember:</p>
+<ul>
```
Comment (spwr007) [inline README.md]: https://guides.github.com/features/mastering-markdown/

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="resources/style.css">
+    <title>Google</title>
```
Comment (spwr007) [inline index.html]: Use 2 tab spaces

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+</head>
+<body>
+    <section class="center">
+        <center>
+            <img src="resources/images/logo.png" alt="company logo" class="company-logo">
```
Comment (spwr007) [inline index.html]: best practise, void elemnts should be self closed, img, input, br, etc, etc

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="resources/style.css">
+    <title>Google</title>
+</head>
+<body>
+    <section class="center">
+        <center>
```
Comment (spwr007) [inline index.html]: mostly avoid styling from tags, such as center, b, i, tag, dont use styling tags.
keep structure and styling separate.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    <span><ion-icon class="icon" name="search-outline"></ion-icon></span>
+                </div>
+                <div class="input-container">
+                    <input type="text" id="searchbox" name="searchbox" class="input-field">
+                </div>
+            </div>
+            <div class="cta">
+                <center>
+                    <input type="submit" value="Google Search">
+                    <input type="submit" value="I'm Feeling Lucky">
```
Comment (spwr007) [inline index.html]: input tags are form elements, should be used in form, without form no submit actions.
use button tags instead or input type button

File: index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+                </div>
+            </div>
+        </center>
+    </section>
+    <footer>
+        <div class="fbar">
+            <div class="country">
+                <span>India</span>
+        </div>
+        <div class="fbar">
```
Comment (spwr007) [inline index.html]: Indentation

File: resources/style.css [hunk 7]
```diff
@@ -0,0 +1,179 @@
+/* General Document Structure */
+
+* {
+    margin: 0;
```
Comment (spwr007) [inline resources/style.css]: 2 tab spaces

File: resources/style.css [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+    padding: 0;
+    box-sizing: border-box;
+}
```
Comment (spwr007) [inline resources/style.css:11]: use reset css, recommended.

File: resources/style.css [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+    height: 33px;
+    font-size: 15px;
+}
+
+.cta {
+    width: 570px;
+    height: 70px;
+}
+
+.cta input[type="submit"] {
```
Comment (spwr007) [inline resources/style.css]: i believe there are only two inputs ;p.

type= 'submit' not needed

File: resources/style.css [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+footer {
+    bottom: 0px;
+    width: 100%;
+    height: auto;
+    position: fixed;
+    background-color: #F2F2F2;
+    color: rgba(0,0,0,.54);
+}
+
+.fbar {
```
Comment (spwr007) [inline resources/style.css]: keep one liner, when block contains only one css prop

File: resources/style.css [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+
+.secondary-nav {
+    float: right;
+    margin-right: 40px;
+    padding: 10px 0;
+}
+
+.secondary-nav li {
+    margin-left: 30px;
+}
```
Comment (spwr007) [inline resources/style.css]: put an endline

**Review by spwr007 — COMMENTED** [review]
@smeetthakkar Reviewed

File: index.html [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+            <div class="footer-nav">
+                <ul class="primary-nav">
+                    <li>Advertising</li>
```
Comment (vishalsg42) [inline index.html]: This should be clickeable. add anchor tag

File: resources/style.css [hunk 13]
```diff
@@ -0,0 +1,179 @@
+/* General Document Structure */
+
```
Comment (vishalsg42) [inline resources/style.css:4]: add comments  for each section as you design

File: index.html [hunk 14]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (vishalsg42) [inline index.html:1]: I understand you have committed all the files once. but try to start the project with git. Also, keep committing codes frequently as when you complete each section.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+<body>
+    <section class="center">
+        <center>
+            <img src="resources/images/logo.png" alt="company logo" class="company-logo">
+            <div class="search-container">
+                <div class="icon">
+                    <span><ion-icon class="icon" name="search-outline"></ion-icon></span>
+                </div>
+                <div class="input-container">
+                    <input type="text" id="searchbox" name="searchbox" class="input-field">
```
Comment (vishalsg42) [inline index.html:20]:
```
<form>
For wrapping form element  use **form-group** classname
  <div class="form-group">
For input tags or form elems use **form-control** classname
       <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Enter email">
  </div>
Submit button should not be wrapped. 
  <button type="submit" class="btn btn-primary">Submit</button>
</form>
```

**Review by vishalsg42 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (smeetthakkar) [reply README.md]: Noted. Thanks for sharing, I'll review and update. The new commit focuses on code related changes, will update the README.md in the next iteration.

File: index.html (hunk 3 above)
Comment (smeetthakkar) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (smeetthakkar) [reply index.html]: Noted. Thanks!

File: index.html (hunk 5 above)
Comment (smeetthakkar) [reply index.html]: Implemented.

File: resources/style.css (hunk 7 above)
Comment (smeetthakkar) [reply resources/style.css]: Updated css files.

File: resources/style.css (hunk 8 above)
Comment (smeetthakkar) [reply resources/style.css:11]: My initial understanding was * {margin: 0, padding: 0, box-sizing: border-box; } resets CSS, however, I have researched a different approach, lmk if that meets your expectations?

File: resources/style.css (hunk 9 above)
Comment (smeetthakkar) [reply resources/style.css]: Haha, coolness ha! Thanks for the tip though!

File: resources/style.css (hunk 10 above)
Comment (smeetthakkar) [reply resources/style.css]: Implemented.

File: resources/style.css (hunk 11 above)
Comment (smeetthakkar) [reply resources/style.css]: Implemented.

File: index.html (hunk 12 above)
Comment (smeetthakkar) [reply index.html]: Implemented.

File: resources/style.css (hunk 13 above)
Comment (smeetthakkar) [reply resources/style.css:4]: Makes total sense, thanks, will keep that in mind in future projects.

File: index.html (hunk 14 above)
Comment (smeetthakkar) [reply index.html:1]: Got it. And, you're right, I agree, having an initial commit on master and then resorting to branches is a good approach. Thanks for the tip!

File: index.html (hunk 15 above)
Comment (smeetthakkar) [reply index.html:20]: Sounds good. I've updated the approach but when I use forms in the future, I'll keep this in mind, thank you!

**Comment by smeetthakkar** [comment]
@vishalsg42 @spwr007 Hi folks! Firstly, thanks a ton for the review!

I've addressed all your comments, please review and lmk your thoughts. FYI, I've update the approach. This means, the markup and styles were rewritten. I've included details in the commit message and the files for your perusal. Even added a tag on the previous commit just in case. 

@spwr007 I need to update the markdown for README.md, that's the only thing remaining. I'll take that up during the next iteration sometime this week. Will keep you posted. Thanks for sharing the resource.

**Review by smeetthakkar — COMMENTED** [review]
All changes reviewed/implemented, please check.

File: resources/style.css (hunk 8 above)
Comment (vishalsg42) [reply resources/style.css:11]: Have a look over this link: [reset css]( https://meyerweb.com/eric/tools/css/reset/)

**Review by vishalsg42 — APPROVED** [review]

File: resources/style.css (hunk 8 above)
Comment (spwr007) [reply resources/style.css:11]: Reset CSS is not only for styling properties, but to maintain the styling along with units across browsers.
The above line does work but not perfectly for all tags. Follow reset css.

Comment (smeetthakkar) [reply resources/style.css:11]: Understood. Vishal shared a link, been using that as well.

File: README.md (hunk 1 above)
Comment (smeetthakkar) [reply README.md]: This is fixed as well. I updated the readme.md file.

**Pull request merged**